libs/backend/src/cast_vote_records/file_system_utils.ts

```diff
@@ -29,11 +29,14 @@ export async function updateCreationTimestampOfDirectoryAndChildrenFiles(
     return entry.name;
   });
   for (const fileName of fileNames) {
+    // Use execFile instead of fs.copyFile because fs.copyFile has issues in prod when the source
+    // and destination paths are both on a USB drive
```

Comment:
I would love to know why this is the case.

Reply:
Yeah, it remains a mystery to me, and Ben, too, who found the issue (https://github.com/votingworks/vxsuite/pull/4262) 🤔